pkg/apis/OWNERS

```diff
@@ -1,44 +1,51 @@
-approvers:
-- erictune
-- lavalamp
-- smarterclayton
-- thockin
-- liggitt
-# - bgrant0607 # manual escalations only
-reviewers:
-- lavalamp
-- smarterclayton
-- wojtek-t
-- deads2k
-- yujuhong
-- brendandburns
-- derekwaynecarr
-- caesarxuchao
-- vishh
-- mikedanese
-- liggitt
-- nikhiljindal
-- gmarek
-- erictune
-- pmorie
-- sttts
-- dchen1107
-- saad-ali
-- luxas
-- janetkuo
-- justinsb
-- pwittrock
-- ncdc
-- tallclair
-- yifan-gu
-- eparis
-- mwielgus
-- feiskyer
-- soltysh
-- piosz
-- dims
-- errordeveloper
-- madhusudancs
-- krousey
-- rootfs
-- jszczepkowski
+".*":
+	approvers:
+	- erictune
+	- lavalamp
+	- smarterclayton
+	- thockin
+	- liggitt
+	# - bgrant0607 # manual escalations only
+	reviewers:
+	- lavalamp
+	- smarterclayton
+	- wojtek-t
+	- deads2k
+	- yujuhong
+	- brendandburns
+	- derekwaynecarr
+	- caesarxuchao
+	- vishh
+	- mikedanese
+	- liggitt
+	- nikhiljindal
+	- gmarek
+	- erictune
+	- pmorie
+	- sttts
+	- dchen1107
+	- saad-ali
+	- luxas
+	- janetkuo
+	- justinsb
+	- pwittrock
+	- ncdc
+	- tallclair
+	- yifan-gu
+	- eparis
+	- mwielgus
+	- feiskyer
+	- soltysh
+	- piosz
+	- dims
+	- errordeveloper
+	- madhusudancs
+	- krousey
+	- rootfs
+	- jszczepkowski
+"^[^/]+/([^/]+/)?types\\.go$": # examples: foo/types.go, foo/bar/types.go
```

Comment:
oh man, these would be way simpler as globs or something - what are the odds of fixing that?